src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -11,29 +12,36 @@ <h1 class="govuk-heading-l">Hearing details</h1>
         </div>
       </dl>
       <br>
-      <h1 class="govuk-heading-m">Hearing stage and result</h1>
-      <span class="govuk-body align-right">
-        <a href="javascript:void(0)" [routerLink]="['/', 'hearings', 'actuals', id, 'hearing-stage-result']"
-           class="govuk-link">Update</a>
-      </span>
-      <dl class="govuk-summary-list">
-        <div class="govuk-summary-list__row">
-          <dt class="govuk-summary-list__key">Hearing stage</dt>
-          <dd class="govuk-summary-list__value">{{hearingTypeDescription}}</dd>
-        </div>
-      </dl>
-      <dl class="govuk-summary-list">
-        <div class="govuk-summary-list__row">
-          <dt class="govuk-summary-list__key">Hearing result</dt>
-          <dd class="govuk-summary-list__value">{{hearingOutcome.hearingResult}}</dd>
+      
+        <h1 class="govuk-heading-m">Hearing stage and result</h1>
+        <span class="govuk-body align-right">
+          <a href="javascript:void(0)" [routerLink]="['/', 'hearings', 'actuals', id, 'hearing-stage-result']"
+            id="hearing-stage-result-update-link"
+            class="govuk-link">Update</a>
+        </span>
+        <div class="govuk-error-message" *ngIf="submitted && hearingResult === ''">
```

Comment:
this (submitted && hearingResult === '') can be extracted to common method and reused in line 25.

Reply:
I have updated the code.